CHANGELOG.md

```diff
@@ -22,6 +22,59 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
+
+  There may be a substantial drop in number of requests observed by subgraphs with this release.
+
+- **Operations can now be made via `GET` requests** ([PR #429](https://github.com/apollographql/router/pull/429))
+
+  The Router now supports `GET` requests for `query` operations.  Previously, the Apollo Router only supported making requests via `POST` requests.  We've always intended on supporting `GET` support, but needed some additional support in place to make sure we could prevent allowing `mutation`s to happen over `GET` requests.
+
+- **Automatic persisted queries (APQ) support** ([PR #433](https://github.com/apollographql/router/pull/433))
+
+  The Router now handles [automatic persisted queries (APQ)] by default, as was previously the case in Apollo Gateway.  APQ support pairs really well with `GET` requests (which also landed in this release) since they allow read operations (e.g., `GET` requests) to be more easily cached by intermediary proxies and CDNs, which typically forbid caching `POST` requests by specification (even if they often are just reads in GraphQL).  Follow the link above to the documentation to test them out.
+
+  [automatic persisted queries (APQ)]: https://www.apollographql.com/docs/apollo-server/performance/apq/
+
+- **New internal Tower architecture and preparation for extensibility** ([PR #319](https://github.com/apollographql/router/pull/319))
+
+  We've introduced new foundational primitives to the Router's request pipeline which facilitate the creation of composable _onion layers_.  For now, this is largely leveraged through a series of internal refactors and we'll need to document and expand on more of the details that facilitate developers building their own custom extensions.  To leverage existing art &mdash; and hopefully maximize compatibility and facilitate familiarity &mdash; we've leveraged the [Tokio Tower `Service`] pattern.
+
+  This should facilitate a number of interesting extension opportunities and we're excited for what's in-store next.  We intend on improving and iterating on the API's ergonomics for common Graph Router behaviors over time, and we'd encourage you to open issues on the repository with use-cases you might think need consideration.
+
+  [Tokio Tower `Service`]: https://docs.rs/tower/latest/tower/trait.Service.html
+
+- **Support for Jaeger HTTP collector in OpenTelemetry** ([PR #479](https://github.com/apollographql/router/pull/479))
+
+  It is now possible to configure Jaeger HTTP collector endpoints within the `opentelemetry` configuration.  Previously, Router only supported the UDP method.
+
+  The [documentation] has also been updated to demonstrate how this can be configured.
+
+  [documentation]: https://www.apollographql.com/docs/router/configuration/#using-jaeger
+
```

Comment:
```suggestion

- **Uplink schema download integration** ([PR #498](https://github.com/apollographql/router/pull/498))

The router can now download its supergraph schema from Studio with Uplink, by setting the `APOLLO_KEY` and `APOLLO_GRAPH_REF` environment variables. This will also set up usage reporting via Spaceport.
```